.circleci/config.yml

```diff
@@ -6,26 +6,30 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4.1-node-browsers
+      - image: circleci/ruby:2.6.1
     working_directory: ~/gapic-generator-ruby
     steps:
       - checkout
+      - run:
+          name: update environment
+          command: |
+            gem update --system
+            gem install bundler -v 1.7.3
+            apt-get install -yqq protobuf-compiler
```

Comment:
What do you think about standardizing on using the `grpc-tools` gem for this, which comes with protoc?

Reply:
I don't know anything about grpc-tools.

Reply:
If we add that gem as a dev dependency can we remove the "Install the Proto Compiler" section of the gapic-generator README?

Reply:
The short version is this:

```bash
$ gem install grpc-tools
$ grpc_tools_ruby_protoc --proto_path=my/protos my/protos/foo.proto
```

In other words, this would allow us to keep everything in Ruby land and install protoc via ruby gems. It also won't interfere with with a local protoc since the binary is named differently.

For a full example you can see how I used it in #45.

The only real downside I can see to this approach is them gem lags new protoc releases a bit, but I really doubt that would ever matter.

Reply:
Yes, that would no longer be needed.

Reply:
The [build with grpc-tools](https://circleci.com/gh/googleapis/gapic-generator-ruby/27) failed, but the [build with apt-get](https://circleci.com/gh/googleapis/gapic-generator-ruby/26) passed.

Reply:
hmm. we have to rename any use of `protoc` to `grpc_tools_ruby_protoc` but that error doesn't look like that's the problem.

Reply:
we likely need to remove stuff like this as well: `--plugin=protoc-gen-ruby-protobuf=/usr/local/bundle/bin/protoc-gen-ruby`, but I'm still confused as to why it would lead to that error.

Reply:
K, removed `apt-get` and switched to using `grpc_tools_ruby_protoc`. The [build passed](https://circleci.com/gh/googleapis/gapic-generator-ruby/28).